### PR TITLE
Re-styled broadcast messages

### PR DESCRIPTION
**Pull request by jgreben**
Re-styled broadcast messages view including re-styled headings, buttonized links next to the heading, and alert-styled message box.

Renamed branch to match issue 206 and committed some changes that @jkeck suggested in closed PR #285.

### CODE REVIEW

**Comment by jgreben** [comment]
@jkeck: I believe that I addressed most if not all of the issues from closed PR #285. I DRY'd up the code, constructed the titles in a hopefully better way, and re-styled the H2's. When you get a chance can you take a look at this PR and let me know if there is anything I missed?

File: app/views/messages/_messages_for_library.html.erb [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
+
+  <% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
+  <% @library = LibraryLocation.library_name_by_code(library_code).to_s %>
+
+  <% if @library.empty? %>
+    <% @library = 'anywhere' %>
+  <% end %>
+
+  <% if messages.none? %>
+    <h2><%= request_type.titleize + " from " + @library %></h2>
```
Comment (jkeck) [inline app/views/messages/_messages_for_library.html.erb]: The best I can tell, this is the only place where the `@library` instance variable is used (I don't think this should be an instance variable w/ the `@`).  Can we just change this to be `<%= request_type.titleize + " from #{library || 'anywhere'}" %>`.  Then we could remove lines 6-8 entirely.

**Comment by jkeck** [comment]
Just one minor comment and I think we're ready to :shipit:

File: app/views/messages/_messages_for_library.html.erb (hunk 1 above)
Comment (jgreben) [inline app/views/messages/_messages_for_library.html.erb]: This does not appear to return 'anywhere' if there is no library returned.

**Pull request merged**